ext/standard/tests/array/array_find_types.phpt

```diff
@@ -0,0 +1,205 @@
+--TEST--
+basic array_find test
```

Comment:
This is probably not accurate.

Reply:
I have adjusted the test description to match `array_map`, `array_filter`, …